config/locales/en.yml

```diff
@@ -296,6 +296,7 @@ en:
         prev_patient: Previous patient
         school: School
         sexual_assault_crisis_org: Sexual assault crisis org
+        title: Referred by
```

Comment:
hm, I think I'd have a hard time guessing what this key referred to, looking at it without the english part. (You can kinda guess from the other keys what the language value is on the other ones you edited here, and I like that a lot.) What do you think about renaming it from `title` to `referred_by`? So this would wind up being:

`referred_by: Referred by`

If we decide to do that we'll want to change it in the spanish too. Let me know what you think!

Reply:
Yeah I was thinking about the best way to do that one. I went with `title: Referred by` because of the preceding heading. That whole section looks like this: 
```
referred_by:
        clinic: Clinic
        crime_victim_advocacy_center: Crime victim advocacy center
        domestic_violence_org: Domestic violence crisis/intervention org
        family: Family member
        friend: Friend
        fund: "%{fund} website or social media"
        homeless: Homeless shelter
        legal_clinic: Legal clinic
        naf: NAF
        nnaf: NNAF
        other_fund: Other abortion fund
        prev_patient: Previous patient
        school: School
        sexual_assault_crisis_org: Sexual assault crisis org
        title: Referred by
        web_search: Google/Web search
        youth: Youth outreach
```

Since the heading of the section is `referred by:` I thought `title` may make sense. In the view it looks like `label: t('patient.helper.referred_by.title')`

This same convention was implemented elsewhere in the yml files. Here are a couple of examples:

```
other_contact:
        name: Other contact name
        phone: Other phone
        relationship: Relationship to other contact
        title: Other Contact
```

```
special_circumstances:
        domestic_violence: Domestic violence
        fetal_diagnosis: Fetal diagnosis
        homelessness: Homelessness
        incest: Incest
        other_medical_issue: Other medical issue
        prison: Prison
        rape: Rape
        title: Special Circumstances
```

I wanted to let you know my reasoning behind my decision and the precedents I based it on. But I am happy to make the change to `referred_by: Referred by` if we think that will be more readable. 

Let me know your thoughts!